### ISSUE TITLE
Crash in ConversationFragment

### ISSUE THREAD

**Issue by arseneHuot** [zendesk/sunshine-conversations-android#76]
Hello,

I got a crash in crashlytics from conversation fragment as you can see below. i use the last release of your sdk. Do you have an idea ?

```
Fatal Exception: java.lang.NullPointerException: uri
       at com.android.internal.util.Preconditions.checkNotNull(Preconditions.java:60)
       at android.content.ContentResolver.query(ContentResolver.java:474)
       at android.content.ContentResolver.query(ContentResolver.java:434)
       at io.smooch.ui.fragment.ConversationFragment$6.run(Unknown Source)
       at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:234)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1113)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:588)
       at java.lang.Thread.run(Thread.java:818)
```

Sorry i dont have more details

Best regards,
Arsène

**Comment by jugarrit** [comment]
We've never seen this crash before, unfortunately. We'll keep an eye on this and let you know if we find anything.

**Comment by DataWearsAHood** [comment]
Hello!

Looks like a few nullpointer crashes have been fixed since this issue was first reported (v5.7.1, v5.15.0 v7.0.2).
Please reopen this issue if you are still experiencing this crash with the latest release.
